src/setupProxy.js

```diff
@@ -10,4 +10,8 @@ module.exports =  function(app) {
     app.use(createProxyMiddleware('/webwork2_files', {
         target: 'http://localhost:3000'
     }));
+
+    app.use(createProxyMiddleware('/work', {
```

Comment:
I think we should add dev configuration, possibly as environment variables if it doesn't affect the build

Not now but in the future we should think through howe we test the lambda in development

I think @Jhuanderson-Rederly said there is a way to run it locally, hopefully in docker

Reply:
I just finished testing it and it didn't seem to work well with the redirect to stage, so I just removed it.